### PR TITLE
Have conda-build test cuml installation

### PR DESCRIPTION
**Pull request by cwharris**
contributes to https://github.com/nv-morpheus/Morpheus/issues/706, as the base container for devcontainer will also be used for the ci container.

### CODE REVIEW

**Comment by cwharris** [comment]
@dagardner-nv in https://github.com/nv-morpheus/Morpheus/pull/758 you added cuml to the development environment. since then, @mdemoret-nv mentioned installing cuml as part of the test environment rather than the development environment, which I've done in this pr. Any thoughts on this?

**Comment by dagardner-nv** [comment]
> @dagardner-nv in #758 you added cuml to the development environment. since then, @mdemoret-nv mentioned installing cuml as part of the test environment rather than the development environment, which I've done in this pr. Any thoughts on this?

I need to rework my PR a bit. We don't need cuml directly in morpheus, but we do need to pin some of our deps such that users are able to install cuml after the fact without breaking the env. I like the idea of having it in the test container.

**Review by dagardner-nv — APPROVED** [review]

File: docker/Dockerfile [hunk 1]
```diff
@@ -146,6 +146,7 @@ RUN apt update && \
         cuda-cupti-dev-${CUDA_MAJOR_VER}-${CUDA_MINOR_VER} \
         cuda-nvml-dev-${CUDA_MAJOR_VER}-${CUDA_MINOR_VER} \
         cuda-nvtx-${CUDA_MAJOR_VER}-${CUDA_MINOR_VER} \
+        cuda-compiler-${CUDA_MAJOR_VER}-${CUDA_MINOR_VER} \
```
Comment (dagardner-nv) [inline docker/Dockerfile:149]: Minor issue, but lets keep the packages sorted

**Review by mdemoret-nv — APPROVED** [review]

**Pull request merged**